### ISSUE TITLE
Python crash when updates come quickly

### ISSUE THREAD

**Issue by pvh** [Gronis/gloomhaven-helper-rfid#2]
Hi Robin! I got a couple hours to work on my project today and managed to reproducibly segfault my app by rapidly sweeping the HP value up and down. Haven't managed to look into why yet but figured I'd start with a quick bug report.

Other than that, I really enjoyed working with the API today and should have something to post up ... as soon as my little Pi Zero manages to finish compiling everything. I'll send something over once I have it going.

**Comment by Gronis** [comment]
That is most likely a buffer overflow. (Data is sent faster than it is deserialized). This is mostly due to a bunch of print commands (that is only necessary when debuging) that slows down deserialization. 

Just for testing if this is the issue, remove all print comments in the deserializer code, or increase the default buffer size (in buffer.hpp).

**Comment by Gronis** [comment]
Is this fixed now?

**Comment by pvh** [comment]
I don't believe so! I'm afraid with the Pandemic my group hasn't met in person since around when I noted the issue.

**Comment by Gronis** [comment]
Ok, I’ll think the parsing messages should be disabled unless built with debug message.
